library/src/main/java/com/novoda/downloadmanager/RoomFile.java

```diff
@@ -14,6 +15,7 @@
 )
 class RoomFile {
 
+    @NonNull
```

Comment:
looks like necessary since you have it in multiple places..

Reply:
Yea this happened when I updated to the latest version of `Room` 😢